examples/example_pump_data_model/transformations/pump_model-populate-pump_container.yaml

```diff
@@ -0,0 +1,26 @@
+externalId: pump_model-populate-pump_container
+name: pump_model-populate-pump_container
+destination:
+  dataModel:
+    space: {{model_space}}
+    externalId: {{data_model}}
+    version: '{{view_Pump_version}}'
+    destinationType: Pump
+  instanceSpace: {{instance_space}}
+  type: instances
+
+# Specify credentials separately like this:
+# You can also use different credentials for the running transformations than the ones you use to deploy
+authentication:
+  clientId: {{clientId}}
+  clientSecret: {{clientSecret}}
+  tokenUri: {{tokenUri}}
+  # Optional: If idP requires providing the scopes
+  cdfProjectName: {{cdfProjectName}}
+  scopes: {{scopes}}
+  # Optional: If idP requires providing the audience
+  audience: {{audience}}
+schedule:
+  # every hour
+  interval: '0 * * * *'
```

Comment:
We are starting to get quite a few places with hardcoded schedules now. Maybe we should add schedules as defaul.config.yaml variables:
`scheduleHourly: '0 * * * *'
scheduleQuarterly: ...`